src/App.css

```diff
@@ -1,38 +1,250 @@
+/* SETUP SNIPPET END */
+
+/* APP STYLES START HERE */
+
+@import url('https://fonts.googleapis.com/css2?family=Anton&family=Archivo+Black&family=Quattrocento&display=swap');
+
+
+/* APP STYLING BEGINS */
 .App {
-  text-align: center;
+  font-size: 125%;
+  max-width: 100vw;
+  margin: 0;
+  padding: 0;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.wrapper {
+  max-width: 1250px;
+  width: 80%;
+  margin: 0 auto;
+  padding: 0;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+
+/* HEADER STARTS HERE */
+header {
+  height: 100vh;
+  width: 100vw;
+  background: url(./assets/header1.jpg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  background-attachment: fixed;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.headerContainer {
+  color: #FFFFFF;
   display: flex;
   flex-direction: column;
+  align-items: flex-start;
+}
+
+h1, h3, h4 {
+  font-family: 'Archivo Black', sans-serif;
+  margin: 0;
+}
+
+h2, button, p, li {
+  font-family: 'Quattrocento', serif;
+  margin: 0.5rem 0;
+}
+
+h1{
+  font-size: 3rem;
+  margin-bottom: 2rem;
+  text-shadow: 3px 4px 6px #0a0a0a;
+  text-transform: uppercase;
+}
+
+h2{
+  text-shadow: 1px -1px 0 #290900;
+}
+
+form {
+  width: 50rem;
+  padding: 0;
+  margin-top: 2rem;
+}
+
+input {
+  width: 25rem;
+  height: 3.5rem;
+  margin: 0 auto;
+  color: #333;
+  background: rgba(255,255,255,0.4);
+  box-shadow: 0 2px 10px 2px rgba(0,0,0,0.5);
+  border-radius: 5px;
+  border: none;
+  outline: none;
+}
+
+button {
+  color: #0a0a0a;
+  width: 10rem;
+  height: 3.4rem;
+  margin: 0 1rem;
+  background: rgba(255, 255, 255,0.75);
+  box-shadow: 0 2px 10px 2px rgba(0,0,0,0.5);
+  border:solid 1px white;
+  border-radius: 5px;
+}
+
+input:hover, input:focus, button:hover, button:focus {
+  background: rgba(255, 255, 255,0.8);
+  box-shadow: 0 6px 10px 6px rgba(0,0,0,0.8);
+} /* HEADER ENDS HERE */
+
+
+/* MAIN STARTS HERE */
+main {
+  display: flex;
+  justify-content: center;
+  align-items: space-around;
+}
+
+.drinkContainer {
+  background: #FFFFFF;
+  box-shadow: rgba(0, 0, 0, 0.25) 0px 54px 55px, rgba(0, 0, 0, 0.12) 0px -12px 30px, rgba(0, 0, 0, 0.12) 0px 4px 6px, rgba(0, 0, 0, 0.17) 0px 12px 13px, rgba(0, 0, 0, 0.09) 0px -3px 5px;
+  border-radius: 0.3rem;
+  width: 90%;
+  margin: 2rem;
+  display: flex;
+  justify-content: space-evenly;  
   align-items: center;
+}
+
+.imgContainer {
+  width: 37.5%;
+  height: 90%;
+  margin: 1.5rem 1rem;
+  border-radius: 0.5rem;
+  display: flex;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  align-items: center;
+}
+
+img {
```

Comment:
I really liked the use of the box-shadow, it gave a very good effect when it shows the results.